dask_planner/src/sql/logical.rs

```diff
@@ -49,28 +50,54 @@ impl PyLogicalPlan {
 
 #[pymethods]
 impl PyLogicalPlan {
```

Comment:
I like the attention to detail and moving the function names to alphabetical order. Also good change on the error handling. We are certainly getting to the stage where paying more attention to little things like that is important.